docs/docs/dev_docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Quickstart
+---
+
+Get started with the Aztec Sandbox
+
+## Introduction
+
+The Aztec Sandbox is an environment for local development on the Aztec Network. It's easy to get setup with just a single, simple command, and contains all the components needed to develop and test Aztec contracts and applications.
+
+### Background
+
+Aztec's Layer 2 network is a fully programmable combined private/public ZK rollup. To achieve this, the network contains the following primary components:
+
+- Aztec Node - Aggregates all of the 'backend' services necessary for the building and publishing of rollups. This packages is currently in development and much of the functionality is mocked.
+- [Aztec RPC Server](https://github.com/AztecProtocol/aztec-packages/tree/master/yarn-project/aztec-rpc) - Normally residing with the end client, this decrypts and stores a client's private state, executes simulations and submits transactions to the Aztec Node.
+- [Aztec.js](https://github.com/AztecProtocol/aztec-packages/tree/master/yarn-project/aztec.js) - Aztec's client library for interacting with the Aztec RPC Server (think Ethers.js). See the getting started guide [here](./aztecjs.md).
+
+All of this is included in the Sandbox, with the exception of Aztec.js which you can use to interact with it.
+
+With the help of Aztec.js you will be able to:
+
+- Create an account
+- Deploy a contract
+- Call view methods on contracts
+- Simulate the calling of contract functions
+- Send transactions to the network
+- Be notified when transactions settle
+- Query chain state such as chain id, block number etc.
+
+This quickstart walks you through installing the Sandbox, deploying your first Noir contract, and verifying its execution!
+
+## Installation
+
+You can run the Sandbox using either Docker or npm.
+
+### With Docker
+
+To install and start the Sandbox paste the line below in a macOS Terminal or Linux shell prompt. You will need to have Docker installed on your machine.
+
+```bash
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+To install a specific version of the sandbox, you can set the environment variable `SANDBOX_VERSION`
+
+```bash
+SANDBOX_VERSION=<version> /bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+NOTE: If `SANDBOX_VERSION` is not defined, the script will pull the latest release of the sandbox.
+
```

Comment:
There's another 'getting started' guide somewhere that mentions port conflicts. We should mention that here too. If they get errors about port conflicts, when running this curl command, they should check whether those ports are already in use (<insert such commands to check>).
To specify different ports, they'd need to edit the docker-compose file which got saved in `.aztec/docker-compose.yml`: `nano .aztec/docker-compose.yml` then change `PORTS: 8080:8080` to `PORTS: <other port>:8080`. (I might have got the docker compose syntax wrong - I'm basing this off memory).

Then, when doing CLI commands, they'd need to specify `--rpc-url=<other port>` with each cli command.

Reply:
Also, if they re-run the `curl` command, it will overwrite their custom changes to the port number in the docker-compose.yml file.

Reply:
Can we open a new issue to change the port from 8080 to 4273 (because it looks like AZTE(c)) and because 8080 is used by everything (especially the mainframe!)

Reply:
I am testing out editing the docker-comose.yml file. It looks like its `<other port>:8080`, but when it starts up it still prints `sandbox Aztec Sandbox JSON-RPC Server listening on port 8080` in the terminal even when it's been changed.

I'll update the quickstart page with this note.

Reply:
created an issue here: https://github.com/AztecProtocol/aztec-packages/issues/2381

I likely wont have time to add this to the docs in the next couple days, but can do later.

Reply:
> but when it starts up it still prints sandbox Aztec Sandbox JSON-RPC Server listening on port 8080

Ah, yeah that'll be because from the sandbox's perspective, it's still communicating via port 8080.